frontend/packages/ceph-storage-plugin/src/components/ocs-install/existing-cluster-modal.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Trans, useTranslation } from 'react-i18next';
+import { match as RouteMatch } from 'react-router';
+import { Button } from '@patternfly/react-core';
+import { Modal } from '@console/shared/src/components/modal';
+import { OCSServiceModel } from '@console/local-storage-operator-plugin/src/features';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src';
+import { history, resourcePathFromModel } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+
+export const ExistingClusterModal: React.FC<ExistingClusterModalProps> = ({
+  match,
+  clusterName,
+  hasStorageCluster,
+}) => {
+  const {
+    params: { ns, appName },
+  } = match;
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
```suggestion
  const [isOpen, setIsOpen] = React.useState(hasStorageCluster);
```

Reply:
I have changed a bit of code now.
This suggestion does not holds now.PTAL again.